source/Makefile.am

```diff
@@ -55,12 +44,12 @@ noinst_HEADERS += input.cpp options.cpp
 # Uninstalled binary maxsimplex
 noinst_PROGRAMS = maxsimplex/maxsimplex
 maxsimplex_maxsimplex_SOURCES = maxsimplex/maxsimplex.cpp
-maxsimplex_maxsimplex_LDADD = libnormaliz.la
+maxsimplex_maxsimplex_LDADD = libnormaliz.la -lgmp -lgmpxx
 
 # Uninstalled binary dynamic
 noinst_PROGRAMS += dynamic/dynamic
 dynamic_dynamic_SOURCES = dynamic/dynamic.cpp
-dynamic_dynamic_LDADD = libnormaliz.la
+dynamic_dynamic_LDADD = libnormaliz.la -lgmp -lgmpxx
```

Comment:
That doesn't seem right -- libnormaliz.la should encode all transitive requirements, precisely so people using it (via GNU libtool) don't need to manually add them.

Reply:
This may in particular actively break code linking against libnormaliz. I.e., looks like a blocker to me; but perhaps there is a rationale, that I then would love to understand

Reply:
If dynamic uses GMP's C++ wrapper through any of the code in the header files it pulls in, it should link against `libgmpxx`. Almost anything you do from `gmpxx.h` means that you use GMP which means that you should link against `libgmp`. That's my understanding how linking works here but I might be wrong and dynamic might actually not use any GMP at all. In that case we would be overlinking here.